larpandoradlcontent/LArDeepLearning/DeepLearningTrackShowerIdAlgorithm.h

```diff
@@ -0,0 +1,50 @@
+/**
+ *  @file   larpandoradlcontent/LArMonitoring/DeepLearningTrackShowerIdAlgorithm.h
+ *
+ *  @brief  Header file for the deep learning track shower id algorithm.
+ *
+ *  $Log: $
+ */
+#ifndef LAR_DEEP_LEARNING_TRACK_SHOWER_ID_ALGORITHM_H
+#define LAR_DEEP_LEARNING_TRACK_SHOWER_ID_ALGORITHM_H 1
+
+#include "Pandora/Algorithm.h"
+
+namespace lar_content
+{
+
+/**
+ *  @brief  DeepLearningTrackShowerIdAlgorithm class
+ */
+class DeepLearningTrackShowerIdAlgorithm: public pandora::Algorithm
+{
+public:
+    /**
+     *  @brief  Default constructor
+     */
+    DeepLearningTrackShowerIdAlgorithm();
+    virtual ~DeepLearningTrackShowerIdAlgorithm();
```

Comment:
Would usually include an empty line between constructor (which has its doxygen info above) and this destructor (which is virtual and doesn't need doxygen info). (Pandora algorithm/process has virtual destructor, of course.)